packages/api/core/helper/dynamic-import.js

```diff
@@ -1,5 +1,23 @@
 const url = require('url');
+const fs = require('fs');
 
 exports.dynamicImport = function dynamicImport(path) {
-  return import(url.pathToFileURL(path));
+  try {
+    return import(fs.existsSync(path) ? url.pathToFileURL(path) : path);
```

Comment:
```suggestion
    return await import(fs.existsSync(path) ? url.pathToFileURL(path) : path);
```

I think because this is async we need to await to catch any errors deep in node